cosign/private/sign.bzl

```diff
@@ -0,0 +1,77 @@
+"Implementation details for sign rule"
+
+_DOC = """Sign an oci_image using cosign binary at a remote registry.
+
+It sings the image by its digest determined beforehand.
```

Comment:
"signs the image"

would be great if it also sang but probably asking too much of Bazel there